RecoTracker/CkfPattern/plugins/GroupedCkfTrajectoryBuilder.cc

```diff
@@ -848,11 +848,11 @@ GroupedCkfTrajectoryBuilder::rebuildSeedingRegion(const TrajectorySeed&seed,
     // (e.g. because no Tracker layers outside seeding region) 
     //
 
-    if ( it->measurements().size()<=startingTraj.measurements().size() ) {
-      rebuiltTrajectories.push_back(std::move(*it));
-      LogDebug("CkfPattern")<< "RebuildSeedingRegion skipped as in-out trajectory does not exceed seed size.";
-      continue;
-    }
+    //if ( it->measurements().size()<=startingTraj.measurements().size() ) {
+    //  rebuiltTrajectories.push_back(std::move(*it));
+    //  LogDebug("CkfPattern")<< "RebuildSeedingRegion skipped as in-out trajectory does not exceed seed size.";
+    //  continue;
+    //}
```

Comment:
If this code is not needed, please remove.
If for some reason it's needed here commented out, please add a clear comment why.
The same for a change in line 1021